src/debugger/rnDebugSession.ts

```diff
@@ -46,14 +46,29 @@ export class RNDebugSession extends LoggingDebugSession {
     private previousAttachArgs: IAttachRequestArgs;
     private rnCdpProxy: ReactNativeCDPProxy | null;
     private cdpProxyLogLevel: LogLevel;
+    private firstDebugSessionStatus: "unused" | "pending" | "used";
```

Comment:
Please don't use strings for such purposes. [`enums`](https://www.typescriptlang.org/docs/handbook/enums.html) are looking more reasonable for that.